lib/galaxy/jobs/runners/__init__.py

```diff
@@ -702,7 +702,7 @@ def monitor(self):
             except Exception:
                 log.exception('Unhandled exception checking active jobs')
             # Sleep a bit before the next state check
-            time.sleep(1)
+            time.sleep(self.app.config.job_runner_monitor_sleep)
```

Comment:
Should probably use theading.Event for everything that runs in a thread, especially if someone sets this to a longer period that 1 sec ? Will create an issue for that.

Reply:
Yeah, I went down that hole a bit and then decided it was more than I wanted to think about for a papercut. I wasn't sure if it was ideal since we are not doing notifications from another thread.

Reply:
We do during shutdown, right ? I've opened https://github.com/galaxyproject/galaxy/issues/12476 for that.